stacks/ecs.yml

```diff
@@ -283,12 +319,33 @@ Resources:
         - !Ref ECSInstanceReturnTrafficSecurityGroup
         - !Ref ECSInstanceOutboundSecurityGroup
       KeyName: !Ref KeyPairName
+      BlockDeviceMappings:
+        - DeviceName: "/dev/xvda"
+          Ebs:
+            DeleteOnTermination: true
+            VolumeSize: 8
+            VolumeType: "gp2"
+        - DeviceName: "/dev/xvdcz"
+          Ebs:
+            DeleteOnTermination: true
+            VolumeSize: !FindInMap [EnvironmentTypeMap, !Ref EnvironmentType, InstanceStorage]
+            VolumeType: "gp2"
       UserData:
-        Fn::Base64: !Sub |
-          #!/bin/bash -xe
-          yum install -y aws-cfn-bootstrap
-          /opt/aws/bin/cfn-init -v --stack ${AWS::StackName} --resource ECSClusterASGLaunchConfiguration --region ${AWS::Region}
-          /opt/aws/bin/cfn-signal -e $? --stack ${AWS::StackName} --resource ECSClusterASG --region ${AWS::Region}
+        Fn::Base64:
+          Fn::Sub:
+            - |
+              Content-Type: multipart/mixed; boundary="==BOUNDARY=="
+              MIME-Version: 1.0
+              --==BOUNDARY==
+              Content-Type: text/cloud-boothook; charset="us-ascii"
+              cloud-init-per once docker_options echo 'OPTIONS="${!OPTIONS} --storage-opt dm.basesize=${BaseSize}G"' >> /etc/sysconfig/docker
```

Comment:
I wanted the literal `OPTIONS=${OPTIONS}` echo'd into that file.  And apparently (according to the docs) CFN will translate `${!Literal}` -> `${Literal}`.